autouri/cli.py

```diff
@@ -79,6 +87,22 @@ def parse_args():
         help='AutoURI(src).rm(): Delete source.',
         parents=[parent_src, parent_all])
 
+    p_rmdir = subparser.add_parser(
+        'rmdir',
+        help='AutoURI(src).rmdir(): Recursively delete all files on '
+        'source directory.',
+        parents=[parent_src, parent_all])
+    p_rmdir.add_argument(
+        '--delete',
+        action='store_true',
+        help='DELETE outputs.')
+    p_rmdir.add_argument(
+        '--nth',
```

Comment:
`nth` is a little hard to read for me. Could have a short option for ease of CLI use and a long one for code readability:

```suggestion
        '-t, --num-threads',
```

Reply:
Yup, fixed as you suggested.